src/openrct2/peep/Peep.cpp

```diff
@@ -2050,8 +2058,8 @@ void peep_thought_set_format_args(const rct_peep_thought* thought)
 {
     set_format_arg(0, rct_string_id, PeepThoughts[thought->type]);
 
-    uint8_t flags = PeepThoughtToActionMap[thought->type].flags;
-    if (flags & 1)
+    PeepThoughtToAction flags = PeepThoughtToActionMap[thought->type].flags;
+    if (flags == PEEP_THOUGHT_ACTION_FLAG_1)
```

Comment:
Although this will work you normally would `&` bit flags. Can you revert that change. (The reason you do this is because often a variable would have more than one flag at the same time. That is not the case this time but its always best to be consistent in the codebase)